overlays/bare-metal-moc/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../base/core/v1/
+  - ../../pipelines/
+  - ../../storage.k8s.io/v1/
```

Comment:
This should be at the top of the list - ensure storageclass exists before doing pv and pvc